storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.3.rst

```diff
@@ -0,0 +1,62 @@
+============================
+ |Percona XtraBackup| 2.4.3
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.3
+on May 23rd 2016. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/>`_ and from :ref:`apt
+<apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the GA (Generally Available) stable release in the 2.4
+series.
+
+New features
+------------
+
+ |Percona XtraBackup| has implemented new
+ :option:`xtrabackup --reencrypt-for-server-id` option. Using this option
+ allows users to start the server instance with different :option:`server_id`
+ from the one the encrypted backup was taken from, like a replication slave
+ or a galera node. When this option is used, |xtrabackup| will, as a prepare
+ step, generate a new master key with ID based on the new :option:`server_id`,
+ store it into keyring file and re-encrypt the tablespace keys inside of
+ tablespace headers.
+
+
+Bugs Fixed
+----------
+
+ Running DDL statements on |Percona Server| 5.7 during the backup process
+ could in some cases lead to corrupted backup. Bug fixed :bug:`1555626`.
+
+ |MySQL| 5.7 can sometimes skip redo logging when creating an index. If such
+ ``ALTER TABLE`` is being issued during the backup, the backup would be
+ inconsistent. Bug fixed :bug:`1582345`.
```

Comment:
We should mention that xtrabackup --backup will now abort with error message if such ALTER TABLE has been done during the backup.